runtime/src/commonMain/kotlin/pbandk/wkt/any.kt

```diff
@@ -40,8 +40,8 @@ private fun Any.protoMergeImpl(plus: Any?): Any = plus?.copy(
 
 private fun Any.protoSizeImpl(): Int {
     var protoSize = 0
-    if (typeUrl.isNotEmpty()) protoSize += pbandk.Sizer.tagSize(1) + pbandk.Sizer.stringSize(typeUrl)
-    if (value.array.isNotEmpty()) protoSize += pbandk.Sizer.tagSize(2) + pbandk.Sizer.bytesSize(value)
+    if (typeUrl.isNotEmpty()) protoSize += pbandk.SizerImpl.tagSize(1) + pbandk.SizerImpl.stringSize(typeUrl)
+    if (value.array.isNotEmpty()) protoSize += pbandk.SizerImpl.tagSize(2) + pbandk.SizerImpl.bytesSize(value)
```

Comment:
Ugh, my intent was to be able to use `Sizer` in all of our code instead of `SizerImpl` (and likewise for `Util` instead of `UtilImpl`). I didn't think about the fact that these are `object`s rather than regular classes, and thus we'd need to refer to the implementations directly. I apologize for suggesting you name them like this.

I think the interfaces are only needed for classes (i.e. `Marshaller` and `Unmarshaller`). Can we continue to declare `Sizer` and `Util` using `expect object Sizer` and `expect object Util`, the same as is currently done on `master`? or does that break something under Kotlin/Native?